app/src/main/java/com/a1ishka/bankaccount/screens/newAccountScreen/SubmitButton.kt

```diff
@@ -0,0 +1,60 @@
+package com.a1ishka.bankaccount.screens.newAccountScreen
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.a1ishka.bankaccount.R
+
+
+@Composable
+fun SubmitButton(
+    accountName: MutableState<String>,
+    accountNumber: MutableState<String>,
+    accountCardNumber: MutableState<String>,
```

Comment:
Why can't you put only String value instead of MutableState? I see you need this parameters only to get value from them.